action_tests/assert.js

```diff
@@ -84,7 +84,57 @@ const EXPECTED_CLI_CALL_FACTORIES = {
       "--upstream",
       process.env.EXPECTED_UPSTREAM,
       "--fix",
-      `--trunk-annotate=12345678`,
+      "--trunk-annotate=12345678",
+    ],
+  ],
+  "pull-request-payload": () => [
+    ["trunk", "version"],
+    ["trunk", "init"],
+    [
+      "trunk",
+      "check",
+      "--ci",
+      "--upstream",
+      process.env.EXPECTED_UPSTREAM,
+      "--github-commit",
+      process.env.EXPECTED_GITHUB_COMMIT,
+      "--github-label",
+      "",
+      "--trunk-annotate=14235603498",
+    ],
+  ],
+  "trunk-merge-payload": () => [
+    ["trunk", "version"],
+    ["trunk", "init"],
+    [
+      "trunk",
+      "check",
+      "--ci",
+      "--upstream",
+      process.env.EXPECTED_UPSTREAM,
+      "--github-commit",
+      process.env.EXPECTED_GITHUB_COMMIT,
+      "--github-label",
+      "",
+      "--trunk-annotate=14235603498",
+    ],
+  ],
+  "all-payload": () => [
+    ["trunk", "version"],
+    ["trunk", "init"],
+    [
+      "trunk",
+      "check",
+      "--all",
+      "--upload",
+      "--series",
+      "main",
+      "--token",
+      process.env.INPUT_TRUNK_TOKEN,
+      '"',
+      '--replace"',
+      ":",
+      '""',
```

Comment:
This doesn't look quite right - I don't think we should have a lone colon as an argument. Is this a bug in the action?

Reply:
see here: https://github.com/trunk-io/trunk-action/pull/168#discussion_r1276333322